streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableFilter.java

```diff
@@ -77,6 +77,7 @@ public void process(K key, Change<V> change) {
             V newValue = computeValue(key, change.newValue);
```

Comment:
Could we add a check on the original `change` value that `newValue` and `oldValue` cannot be both null?